packages/next/build/webpack-config.ts

```diff
@@ -236,6 +237,7 @@ export default async function getBaseWebpackConfig(
         default: false,
         vendors: false,
         framework: {
+          chunks: 'all',
```

Comment:
Let's add a comment explaining what this does.

Reply:
Done. This is on my short list of further optimizations to work on over the next week or two